ci-images/Dockerfile.base

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf install -y 'dnf-command(config-manager)' && \
+    dnf config-manager --set-enabled crb && \
+    dnf install -y git unzip make gcc which nmstate-devel
+
+COPY --from=registry.ci.openshift.org/openshift/release:golang-1.17 /usr/local/go /usr/local/go
+
+ENV GOPATH=/go
+ENV GOROOT=/usr/local/go
+ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+
+COPY . /assisted-service/
+WORKDIR /assisted-service/
+
+ENV CGO_CFLAGS="-g -O2 -Wno-return-local-addr"
```

Comment:
This got removed in https://github.com/openshift/assisted-service/pull/4393 , should we do the same here?

Reply:
yes it should
I'll shortly sync all code 
thank you!

Reply:
wait.. this has not been removed. I can still see it here: https://github.com/openshift/release/blob/88f92b0451149c66eca8ae65562e249647e8aa21/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L80

Reply:
I can remove it if it makes sense to do so also for the CI environment, but as-is it's running okay somehow

Reply:
I think it should be removed. Otherwise, developers with a host that has a newer version of Glibc won't be able to work with this.